plutus-core/plutus-core/src/PlutusCore/Evaluation/Machine/CostingFun/Core.hs

```diff
@@ -317,11 +317,11 @@ data ModelTwoArguments =
   | ModelTwoArgumentsLinearInX          OneVariableLinearFunction
   | ModelTwoArgumentsLinearInY          OneVariableLinearFunction
   | ModelTwoArgumentsLinearInXAndY      TwoVariableLinearFunction
-  | ModelTwoArgumentsAddedSizes         ModelAddedSizes
+  | ModelTwoArgumentsAddedSizes         OneVariableLinearFunction
   | ModelTwoArgumentsSubtractedSizes    ModelSubtractedSizes
-  | ModelTwoArgumentsMultipliedSizes    ModelMultipliedSizes
-  | ModelTwoArgumentsMinSize            ModelMinSize
-  | ModelTwoArgumentsMaxSize            ModelMaxSize
+  | ModelTwoArgumentsMultipliedSizes    OneVariableLinearFunction
+  | ModelTwoArgumentsMinSize            OneVariableLinearFunction
+  | ModelTwoArgumentsMaxSize            OneVariableLinearFunction
```

Comment:
Drop the definitions of `ModelAddedSizes`, `ModelMultipliedSizes`, `ModelMinSize` and `ModelMaxSize` then?

Reply:
Yes, I want to rework some of this later.  It's a bit confusing because it'll change the names of some of the cost model parameters and it's kind of orthogonal to what I was trying to (although I did get carried away and make some changes along those lines that probably shouldn't be in this PR).